### PR TITLE
UserGuide: Update until example to -until 21/10/2017

### PR DESCRIPTION
**Pull request by dlworldpeace**
Fix [#444](https://github.com/reposense/RepoSense/issues/444)
```
In verifying the `Customize Using Command Line Parameters` part of User Guide, 
the example for `until` parameter given is `-since 01/10/2017`.

However, it is a mistake as the example should be updated to `-until 01/10/2017`
in order to function as an `until` parameter.

Let's correct this mistake in the User Guide instructions.
```

### CODE REVIEW

**Comment by ongspxm** [comment]
resolve conflict pls, thxs

**Comment by yong24s** [comment]
@dlworldpeace 

Please update PR title and proposed commit message, you can refer to this [PR](https://github.com/reposense/RepoSense/pull/416).

**Comment by dlworldpeace** [comment]
> @dlworldpeace
> 
> Please update PR title and proposed commit message, you can refer to this [PR](https://github.com/reposense/RepoSense/pull/416).

updated, ready for review.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ In addition, there are some _optional_ extra parameters you can use to customize
 * **`-since START_DATE`**: The start date of analysis. Format: `DD/MM/YYYY`<br>
   Example:`-since 21/10/2017`
 * **`-until END_DATE`**: The end date of analysis. The analysis excludes the end date. Format: `DD/MM/YYYY`<br>
-  Example:`-since 21/10/2017`
+  Example:`-until 01/10/2017`
```
Comment (yamidark) [inline docs/UserGuide.md]: Keep the date as `21/10/2017`, as it is intended to prevent confusion on whether the date is `DD/MM` or `MM/DD` format

**Pull request merged**